README.md

```diff
@@ -162,7 +163,11 @@ Merges the Dublin Core metadata into the hOCR file by encoding the data in its h
 
 ### hocr-pdf
 
-TODO
+```
+Usage: hocr-pdf <imgdir>
+```
+
+Create a searchable PDF from a pile of hOCR and JPEG
```

Comment:
We already have this from the last commit: https://github.com/tmbdev/hocr-tools/commit/cbac14b4a3a5f5d37c73a19f4e6a4dad875ca3f5